app/models/lookup_keys/variable_lookup_key.rb

```diff
@@ -9,7 +9,7 @@ class VariableLookupKey < LookupKey
   scoped_search :relation => :puppetclass, :on => :name, :complete_value => true, :rename => :puppetclass
 
   def editable_by_user?
-    VariableLookupKey.authorized(:edit_external_variables).where(:id => id).exists?
```

Comment:
Aren't these also used by smart class parameters? Will this mean that any user can edit those then?

Reply:
no, they aren't. variable lookup key is smart variables only. This is a step in the process of removing them completely.

Reply:
Normally I'd say that permissions should be removed last. Given the permission no longer exists, no user should have it. In other words, it shouldn't editable anymore. In other words, the method should return `false` rather than doing a DB query.

Reply:
if we were keeping the model i'd agree, but considering this whole model will be gone within a few days, i don't think it's worth bothering with right now.